lib/common_fw/common_fw.cpp

```diff
@@ -36,36 +41,34 @@ bool write_to_SD(std::string message, const char * file_name) {
  * Constructs packet in the following format:
  * {<sensor_ID>,<data1>,<data2>, ...,<dataN>|checksum}
  */
-String make_packet(int id, bool error) {
-  String packet_content = (String)id;
-  packet_content += ",";
+uint8_t make_packet(uint8_t id, bool error) {
+  int i = 0;
+  packet[i++] = (PACKET_START >> 8) & 0xFF;
+  packet[i++] = PACKET_START & 0xFF;
```

Comment:
Does this send the PACKET_START byte forwards and backwards?

Reply:
?? Its big endian.
I'm not sure what you're asking.

Reply:
Yeah, sry. I made this comment when I saw the start_packet and end_packet variables were only 1 byte instead of 2 so I wasn't sure what u were doing here. I understand now tho